gap/tools/display.gi

```diff
@@ -727,7 +727,7 @@ function(S, opts)
 
   # TODO make PartialOrderOfDClasses return a digraph
   rel := OutNeighbours(DigraphReflexiveTransitiveReduction(
-                       Digraph(PartialOrderOfDClasses(S))));
+                       DigraphNC(PartialOrderOfDClasses(S))));
   for i in [1 .. Length(rel)] do
```

Comment:
@james-d-mitchell I'm happy to remove this. I made it simply to speed up `DotString`, since from looking at the code it seemed almost impossible that `PartialOrderOfDClasses` could return something which was not a digraph.